src/videosubscriberaccount.cs

```diff
@@ -103,10 +103,12 @@ interface IVSAccountManagerDelegate { }
 	interface VSAccountManagerDelegate {
 
 		[Abstract]
+		[NoMac]
```

Comment:
This is a breaking change, it shows up as such in the `API & Generator diff`

Note that it seems you'll have to download the diff from Azure DevOps and hunt through the files to find it, because the links here don't work: https://xamarin.github.io/macios.ci/pr/PR11977/4890030/apicomparison/mac-api-diff.html

I think the best you can do is to add an `[Obsoleted (...)]` attribute, and hardcode an exception in introspection if there are any errors.

CC @mandel-macaque ^

Reply:
@rolfbjarne So I made these changes https://gist.github.com/tj-devel709/24bf5f179de2cb09184b6946f229637b based on other examples I found. Introspection does not seem to complain but Xtro is saying:
```
!extra-protocol-member! unexpected selector VSAccountManagerDelegate::accountManager:dismissViewController: found
!extra-protocol-member! unexpected selector VSAccountManagerDelegate::accountManager:presentViewController: found
```
Would I also add this to the ignore with a comment about marking it obsoleted?

Reply:
@rolfbjarne those links should work, is there a diff between API diff and Generator diff? I might have been cleaning too many files..

Reply:
@mandel-macaque open https://xamarin.github.io/macios.ci/pr/PR11977/4890030/apicomparison/mac-api-diff.html and try to click on any of the links there. Nothing happens.

Reply:
@tj-devel709 yes, this warrants an ignore addition to xtro.

Reply:
@rolfbjarne Okay, updated!

Reply:
I'm also not really sure how to read the Generator Diff.. Looks like the last iteration may have been complaining about me adding the NoMacCatalyst at certain places?